week-2/implement/lookup.js

```diff
@@ -1,17 +1,12 @@
-function createLookup() {
+function createLookup(array) {
   // implementation here
+  let object = {};
+  for (const element of array) {
+    const [country, currency] = element;
```

Comment:
Excellent use of destructuring here Areeb

Reply:
Thank you!